js/main.js

```diff
@@ -35,6 +42,18 @@ Site = {
   onResize: function() {
     var _this = this;
 
+    _this.resizeDelay();
+  },
+
+  resizeDelay: function() { 
```

Comment:
and whatever happened to KISS. this code is more legible than what irish posted and just what we need for this purpose.